modtools/dynamic-adv-reactions.lua

```diff
@@ -0,0 +1,892 @@
+Registered reactions are cleared when the world is unloaded, and so it is best to include any commands for this within an ``onLoad*.init`` file.
+Checks with multiple instances of the same type of condition are possible if registered via code rather than with commands.
+
+* ``-reaction reactionCode``
+  (Required)
+  Specify the reaction the conditions will allow.
+  Examples::
+
+    MAKE_SHARP_ROCK
+    "ASSEMBLE STONE AXE"
+
+Conditions:
+Most conditions can optionally be a table of conditions, rather than a single argument. That condition is satisfied if any of them are met.
+
+* ``-creature token``
+  Require the adventurer to be the specified creature.
+  Can optionally be a table of arguments.
+  Example: DWARF
+
+* ``-caste token``
+  Require the adventurer to be the specified caste.
+  Can optionally be a table of arguments.
+  Examples::
+
+    FEMALE
+    DRONE
+
+* ``-creatureCaste tokens``
+  Require the adventurer to be the specified creature + caste combination.
+  Can optionally be a table of arguments.
+  Example: DWARF:MALE
+
+* ``-creatureClass class``
+  Require the adventurer to be a creature with the specified creature class.
+  Can optionally be a table of arguments.
+  Examples::
+
+    GENERAL_POISON
+    MAMMAL
+
+* ``-sphere sphere``
+  Require the adventurer to be a creature with the specified sphere.
+  Can optionally be a table of arguments.
+  Examples::
+
+    METALS
+    MUCK
+
+* ``-syndrome name``
+  Require the adventurer to be under the effects of the specified syndrome.
+  Can optionally be a table of arguments.
+  Examples::
+
+    "giant cave spider bite"
+    "night sickness"
+
+* ``-syndromeClass class``
+  Require the adventurer to be under the effects of a syndrome with the specified SYN_CLASS.
+  Can optionally be a table of arguments.
+  Example: RAISED_UNDEAD
+
+* ``-syndromeIdentifier identifier``
+  Require the adventurer to be under the effects of a syndrome with the specified SYN_IDENTIFIER.
+  Can optionally be a table of arguments.
+  Example: INEBRIATION
+
+* ``-secret id``
+  Require the adventurer to know the specified secret.
+  Can optionally be a table of arguments.
+  Since game-generated secrets have unique names, this is more likely for use with mod-made secrets, which keep the same ids between worlds.
+
+* ``-item token``
+  Require the adventurer to be holding the specified item in their hands.
+  Can optionally be a table of arguments.
+  Examples::
+
+    WEAPON:ITEM_WEAPON_PICK
+    BUCKET:NONE
+
+* ``-itemType token``
+  Require the adventurer to be holding the an item of the specified item type in their hands.
+  Can optionally be a table of arguments.
+  Examples::
+
+    WEAPON
+    COIN
+    BOX
+
+``-moon [ startPhase endPhase ]``
+  Require the moon to be within a set bound of phases.
+  See the wiki entry for what numbers correspond to which phases: https://dwarffortresswiki.org/index.php/DF2014:Syndrome#MOON_PHASE
+
+Example Usage:
```

Comment:
You generally need a double-newline if you want things to display on a separate line. It looks like this now:
![image](https://user-images.githubusercontent.com/3719547/102706367-4b71af00-425f-11eb-9edd-bd8e253c3ba3.png)

You could also make this a subheading by underlining it with `-----` (matching the length of the text) or bold by surrounding it with `**`:
```
Example Usage
-------------
```
or
```
**Example Usage:**

...
```

Reply:
I've side-stepped the issue by making those two examples bulleted instead :P